src/App.js

```diff
@@ -1,22 +1,19 @@
-import { useState } from "react";
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import { useQuery } from "react-query";
+import { useState } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useQuery } from 'react-query';
 
 import {
```

Comment:
this page is missing some formatting...there are ```;``` and the import from pages should be inline which is interesting.